pkg/api/defaults.go

```diff
@@ -218,7 +218,8 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 				}
 				// ipv4 and ipv6 subnet for dual stack
 				if cs.Properties.FeatureFlags.IsFeatureEnabled("EnableIPv6DualStack") {
-					o.KubernetesConfig.ClusterSubnet = strings.Join([]string{DefaultKubernetesClusterSubnet, DefaultKubernetesClusterSubnetIPv6}, ",")
+					defaultKubernetesClusterSubnetIPv6 := cs.getDefaultKubernetesClusterSubnetIPv6()
```

Comment:
I think we can just invoke `cs.getDefaultKubernetesClusterSubnetIPv6()` in L222, we don't need the `defaultKubernetesClusterSubnetIPv6` variable

Reply:
updated!